### PR TITLE
Added a test for the Registration and Password Recovery API with Phone number.

### PR DESCRIPTION
**Pull request by Vikhthor**
Tests the registration and password recovery API endpoints.

### CODE REVIEW

**Review by Douglasokolaa — COMMENTED** [review]
Properly name your Pull Request so we know its purpose

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# apitester
```
Comment (Douglasokolaa) [inline README.md:1]: Why was this deleted?

Comment (Vikhthor) [reply README.md:1]: OK

Comment (Vikhthor) [reply README.md:1]: Though the testREADME.md file describes the test

**Comment by Vikhthor** [comment]
Please do you need me to make a new Pull Request?

**Comment by Clish254** [comment]
> Tests the registration and password recovery API endpoints.
@Vikhthor nice work however the logs from the second test on successful operation are kinda misleading, you print "Recovery email sent successfully" but the reset token is sent to a phone number since you sent a phone number in the request. It would be nice if you also test user registration and password recovery with email but I understand the user registration endpoint docs doesn't tell you whether a user can register with an email.

**Comment by Vikhthor** [comment]
Oh yes. It should have been "Recovery SMS sent successfully" thanks.

**Comment by Clish254** [comment]
> Oh yes. It should have been "Recovery SMS sent successfully" thanks.

Cool, fix that and push tests for user registration with email and password recovery with email. To register a user with their email address just send their email address with a field called `email`.

**Comment by Vikhthor** [comment]
Ok

**Comment by Clish254** [comment]
Tests for user registration and password recovery with email and phone number should be separate.

**Comment by Vikhthor** [comment]
Done

**Comment by Clish254** [comment]
> Done

Are you sure? I see only two tests

**Comment by Vikhthor** [comment]
"Added a test for the Registration and Password Recovery API with Phone number" tests the registration and password recovery using the user's phone number.
"Added a test for the Registration and Password Recovery API with Email" tests the registration and password recovery using the user's email.

**Comment by Vikhthor** [comment]
They are both on separate Pull requests

**Comment by Clish254** [comment]
> They are both on separate Pull requests

Okay, seen

**Review by Clish254 — APPROVED** [review]

**Comment by Vikhthor** [comment]
Thanks
